lib/formats/dotnet-core-parser.js

```diff
@@ -76,14 +75,10 @@ function parse(fileContent, packageTree) {
   });
 
   // restructure tree with "from" chains.
-  function buildPath(node, from) {
-
-    node.from = node.from.concat(from);
+  function buildPath(node) {
```

Comment:
does this function do anything now after removing these lines?

Reply:
nope